#  Name the experiment. This name will be the name of the report
experiment_name: "dynamic_obstacles_scenario_report"

#Save directory of results
results_directory: ''

#  Add the world file to benchmarking_tool/world and provide the name of the new file
world_name: 'dynamic_obstacles_world.world'

#  Add the map files to benchmarking_tool/map and provide the name of the new yaml fi
map_name: 'map.yaml' 

#  Add gazebo model path, if multiple path, please spereate them by a :
models_path: ''

#  Choose one of the following controllers, by adding its name to the controller type list
#  Note that these are the avaible controllers for the provided example
#  DWB
#  RPP
#  RPP_RSC
#  DWB_RSC
controller_type: ['DWB', 'DWB_RSC']   #name of behaviour tree 

#  This option is intended for testing the same controller several times
#  if the user do not need this test, set it to zero 
trails_num: 10

#  Add the absolute path of the robot navigation configuration
nav_config: '/home/riot/src/ros2_ws/src/ROSNavBench/turtlebot3/turtlebot3_navigation2/param/waffle.yaml'

#  Add the directory of the behaviour tree
behaviour_tree_directory: "/home/riot/src/ros2_ws/src/ROSNavBench/turtlebot3/turtlebot3_navigation2/param"

#  Add the absolute path of the urdf and model files inside single qoutes ''
urdf_file: '/home/riot/src/ros2_ws/src/ROSNavBench/turtlebot3/turtlebot3_description/urdf/turtlebot3_waffle.urdf'

#  If the robot is spawned by only urdf, then set the model_file to 'None'
model_file: '/home/riot/src/ros2_ws/src/ROSNavBench/turtlebot3_simulations/turtlebot3_gazebo/models/turtlebot3_waffle/model.sdf'

#  Provide the pose in a decimal form
spawn_pose_x: 4.0
spawn_pose_y: -4.0
spawn_pose_yaw: 1.58   #radians 


#  Specify either to send one goal, specific trajectory, or several waypoints 
#  square, circle, several_waypoints, one_goal
trajectory_type: 'one_goal'

#  One goal parameter 
goal_pose_x: 7.3
goal_pose_y: 2.6
goal_pose_yaw: 1.58   #radians 

